source/cpp/include/focusrite/e2e/ComponentSearch.h

```diff
@@ -7,16 +7,14 @@ namespace focusrite::e2e
 class ComponentSearch
 {
 public:
-    static constexpr char testId [] = "test-id";
-    static constexpr char windowId [] = "window-id";
-
     static juce::Component * findWithId (const juce::String & componentId, int skip = 0);
     static juce::TopLevelWindow * findWindowWithId (const juce::String & windowId = {});
 
     static int countChildComponents (const juce::Component & parent,
                                      const juce::String & matchingId);
 
     static void setTestId (juce::Component & component, const juce::String & id);
+    static void setWindowId (juce::TopLevelWindow & window, const juce::String & id);
```

Comment:
Maybe I'm a little bit confused, but if `setWindowId` didn't exist before now, then how did anything ever get set with the `window-id` id? I see a find for it but no obvious way to set it. Is it just supposed to be an internal construct that users of the library shouldn't interact with?

Reply:
I think in Studio we would have used something like `window.getProperties().setWithDefault (focusrite::e2e::ComponentSearch::windowId, "studio-window-id")`.